samples/interactive-tutorials/user_environment_setup.sh

```diff
@@ -61,3 +61,4 @@ gcloud auth activate-service-account --key-file ~/key.json
 # install needed Google client libraries
 cd ~/cloudshell_open/nodejs-retail/samples
 npm install
+'success'
```

Comment:
will it call a function `success()`?

Reply:
yes, it is calling success() function, how i tested it was working